core/contracts/financial-templates/implementation/WETH9.sol

```diff
@@ -55,8 +55,8 @@ contract WETH9 {
     function withdraw(uint256 wad) public {
         require(balanceOf[msg.sender] >= wad);
         balanceOf[msg.sender] -= wad;
-        msg.sender.transfer(wad);
         emit Withdrawal(msg.sender, wad);
+        msg.sender.transfer(wad);
```

Comment:
This is a re-entrancy fix, complaining that emitting the event `Withdrawal` is an "effect" in the "check-effects-interaction" paradigm that should be before the `transfer()`

Reply:
Can we ignore this file? This isn't our code -- we should probably leave it as it is in the real WETH contract, no?

Reply:
This is verified WETH9 code: https://etherscan.io/address/0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2#code

I'll just ignore this contract as suggested